levels/level_manager_test.go

```diff
@@ -2082,3 +2082,173 @@ func TestRegisterDeadVersionRangeDedup(t *testing.T) {
 
 	require.Equal(t, OverlappingTables(expected), tables)
 }
+func TestContainsTable(t *testing.T) {
+	populatedSeg := &segment{
+		format: 26,
+		tableEntries: []*TableEntry{
+			{
+				SSTableID:  []byte("sstableid1"),
+				RangeStart: []byte("rangestart1"),
+				RangeEnd:   []byte("rangeend1"),
+			},
+			{
+				SSTableID:  []byte("sstableid2"),
+				RangeStart: []byte("rangestart2"),
+				RangeEnd:   []byte("rangeend2"),
```

Comment:
The ranges in these two tables are overlapping - that would normally only be the case for L0. I'd suggest testing L0 (overlapping table entries) and L > 0 (no overlapping separately).

Reply:
I ended up deleting this test since the binary search approach for `containsTable` wasn't valid